resolwe/elastic/tests/test_app/elastic_indexes.py

```diff
@@ -14,13 +14,18 @@ class TestSearchDocument(BaseDocument):
     num = dsl.Integer()
     json = dsl.Object()
 
+    field_name = Name()
+    field_process_type = ProcessType()
+
     class Meta:
         index = 'test_search'
 
 
 class TestSearchIndex(BaseIndex):
     mapping = {
         'num': 'number',
+        'field_name': 'name',
+        'field_process_type': 'name',
```

Comment:
Will be much more clear if you add `type` or `process_type` field instead of just using `name`.